requirements/base.in

```diff
@@ -6,7 +6,7 @@ Django>=1.11
 django-model-utils
 djangorestframework
 django-ipware>=1.1.0
-jsonfield>=2.0.2
+jsonfield2==3.0.3
```

Comment:
The version constraint should move to `requirements/constraints.txt` with a comment explaining why it is needed.  Just leave a non-constrained `jsonfield2` entry here to represent the direct use of the package.

Reply:
@jmbowman We have `jsonfield2` at 2 places, unpinned in `base.in` and pinned in `constraints.txt`. And now this is causing `make upgrade` failure in `edx-enterprise` because upgrade tries to pull the latest version(3.1.0) of `jsonfield2` and `3.1.0` requires `Django>=2.2`. https://github.com/rpkilby/jsonfield2/blob/master/setup.py#L15

As per my understanding we should not have `jsonfield2` in `base.in`.

Any thoughts?

Reply:
This is being sorted out in https://github.com/edx/edx-enterprise/pull/689 ; I'm about to start reviewing that, you're welcome to also take a look and figure out if it addresses your concerns.